configuration/overview.mdx

```diff
@@ -80,6 +80,20 @@ These properties are as follows:
 | tracing.jaeger.host | The UDP host destination to report spans | localhost | v0.17.0 |
 | tracing.jaeger.port | The UDP port destination to report spans | 6831      | v0.17.0 |
 
+### Audit Events
+
+| Property                 | Description                                     | Default | Since   |
+| ------------------------ | ----------------------------------------------- | ------- | ------- |
+| audit.buffer.capacity    | Max capacity of buffer to send events to sinks  | 2       | v1.21.0 |
+| audit.buffer.flushPeriod | Duration to wait before sending events to sinks | 2m      | v1.21.0 |
+
+### Audit Events: Log File
+
+| Property                 | Description                                | Default        | Since   |
+| ------------------------ | ------------------------------------------ | -------------- | ------- |
+| audit.sinks.log.enabled  | Enable log file sink                       | false          | v1.21.0 |
+| audit.sinks.log.path     | File path to send audit events to          | $(PWD)/log.txt | v1.21.0 |
```

Comment:
This should likely default to somewhere in `/var/log` ie `/var/log/flipt/audit.json` and end in `.json` I think